adapter/serviceControl/client.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceControl
+
+import (
+	"net/http"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	servicecontrol "google.golang.org/api/servicecontrol/v1"
+
+	"istio.io/mixer/pkg/adapter"
+)
+
+type client interface {
```

Comment:
Could this just be a function pointer type instead of an interface? That would seem cleaner to me. memQuota uses this approach and I think a few other adapters too.